druntime/src/core/interpolation.d

```diff
@@ -130,7 +134,7 @@ struct InterpolatedLiteral(string text) {
         segment of the tuple, for easy access and compatibility with
         existing functions like `std.stdio.writeln` and `std.conv.text`.
     +/
-    string toString() const @nogc pure nothrow @safe {
+    static string toString() @nogc pure nothrow @safe {
```

Comment:
How about:

```suggestion
    enum toString = text;
```

If it's not the case already, we should add support for it in `std.format`.

Alternatively, why couldn't you do:
```d
alias toString = __getEmptyString;
```
here as well?

Reply:
Yes I’ve thought of that too. It doesn’t work in current Phobos. 

This can’t be empty string or it will not print the right thing when passing to writeln.